ray-operator/controllers/raycluster_controller.go

```diff
@@ -242,7 +242,41 @@ func (r *RayClusterReconciler) reconcilePods(instance *rayiov1alpha1.RayCluster)
 				return err
 			}
 		}
+	} else {
```

Comment:
@wilsonwang371 Can you help add a feature flag for this feature?  similar like what autoscaler did https://github.com/ray-project/kuberay/blob/8486e82ea60a9605bb692dcedbda28f302981bac/ray-operator/main.go#L55-L56

We didn't get enough feedback from community and let's add the protection just in case someone reports the issue. WDYT?

Reply:
@wilsonwang371 kindly ping. Let's add the flag and merge the PR?

Reply:
lets wait some time. I am working on a new patch